src/dropdown.test.aria.ts

```diff
@@ -29,24 +29,24 @@ test('disabled=${false}', async ({ page }) => {
   `);
 });
 
-test.describe('filterable', () => {
+test.describe('query("o")', () => {
```

Comment:
A deviation from our current test name syntax. But this is the closest I was able to get to it while covering multiple filtering states.